README.md

```diff
@@ -25,6 +25,7 @@ To get started, first, [obtain an API token](https://appcenter.ms/settings/apito
 
 ```ruby
 appcenter_fetch_devices(
+  owner_type: "user", # Default is user - set to organisation for appcenter organisations
```

Comment:
It's `organization` not `organisation`. I would also place this line right after `owner_type`.

Reply:
organisation propper english spelling! I'm sorry chap!

Reply:
this has been updated

Reply:
this has been updated